server/tests/report.test.js

```diff
@@ -0,0 +1,158 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import dotenv from 'dotenv';
+import app from '../index';
+import reportData from './testData/report.data';
+
+
+dotenv.config();
```

Comment:
You imported dotenv mistakenly bro. Check it out.

Reply:
Okay. Thanks for pointing this out. I will fix it